src/http/input/conditions/BasicConditionsParser.ts

```diff
@@ -58,6 +59,7 @@ export class BasicConditionsParser extends ConditionsParser {
    * Undefined if there is no value for the given header name.
    */
   private parseTagHeader(request: HttpRequest, header: 'if-match' | 'if-none-match'): string[] | undefined {
-    return request.headers[header]?.trim().split(/\s*,\s*/u);
+    const headerValue = request.headers[header];
+    return headerValue ? splitCommaSeparated(headerValue.trim()) : undefined;
```

Comment:
```suggestion
    if (headerValue) {
      return splitCommaSeparated(headerValue.trim());
    }
```
It's a short function so I think we can fully write out the ternary operator.